src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalValue = days * 40;
+
+  if (days >= 3 && days < 7) {
+    totalValue = days * 40 - 20;
```

Comment:
There is a redundant calculation of 'days * 40' here which was already done in line 7. You can use 'totalValue' instead of recalculating it.